app/assets/javascripts/invitations.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function() {
+    $('form#new_invitation').submit(function() {
+        //console.log('TEST');
+        var valuesToSubmit = $(this).serialize();
+        $.ajax({
+            type: "POST",
+            url: ('invitations'),
+            data: valuesToSubmit,
+            dataType: "JSON"
+        }).success(function (json) {
+            //console.log("success", json);
+            var messageTextArea = document.getElementById('message');
+            messageTextArea.innerText = json.recipient_email + " was invited to your team";
+            messageTextArea.style.border = "thick double #7797db";
+            var pendingInv = document.getElementById('pending-invitations');
+            pendingInv.innerHTML = pendingInv.innerHTML + json.recipient_email;
+            $("#invitation_recipient_email").val('');
+        }).error(function (json) {
+            //console.log("error", json);
+            var messageTextArea = document.getElementById('message')
+            messageTextArea.innerText = json.responseJSON[0];
```

Comment:
hmm.. I don't understand what's going on with these errors numbers... in console I have 422.